### PR TITLE
test!: add tests for KeyPairBuilder struct

### PR DESCRIPTION
**Pull request by amilumibe**

### CODE REVIEW

File: coins/ethereum/src/blockchain_client.rs [hunk 1]
```diff
@@ -0,0 +1,91 @@
+
```
Comment (devinpearson) [inline coins/ethereum/src/blockchain_client.rs:1]: This blockchain_client should not be in a change marked as additional tests for KeyPairBuilder.

File: walletd/src/key_pair.rs [hunk 2]
```diff
@@ -34,11 +34,16 @@ pub enum MnemonicKeyPairType {
 /// This struct is used specify options for and build a [KeyPair] struct
 #[derive(Debug, Clone, PartialEq, Eq, Default)]
 pub struct KeyPairBuilder {
-    mnemonic_phrase: Option<String>,
-    mnemonic_seed: Option<Seed>,
-    passphrase: Option<String>,
-    network_type: HDNetworkType,
-    style: MnemonicKeyPairType,
+    /// Option to set the mnemomonic phrase
```
Comment (devinpearson) [inline walletd/src/key_pair.rs:37]: These should not be made public to satisfy integration tests. they should be tested via unit tests where the values are accessible as private properties

**Review by devinpearson — CHANGES_REQUESTED** [review]

**Review by karlbuys — COMMENTED** [review]
Barring the comments by Devin, it looks good to me

File: walletd/src/key_pair.rs (hunk 2 above)
Comment (amilumibe) [reply walletd/src/key_pair.rs:37]: I can make them private again but then would need to add the methods for getting the values

**Review by devinpearson — APPROVED** [review]

**Pull request merged**